docs/develop/requirements-property-unified-manifest.md

```diff
@@ -0,0 +1,197 @@
+
+## extensions.requirements
+
+The "extensions.requirements" property specifies the scopes, form factors, and [requirement sets](/javascript/api/requirement-sets) for Microsoft 365 Add-ins. If the Microsoft 365 version doesn't support the specified requirements, then the extension won't be available for installation. Users won't see it in the Office UI for searching and installing add-ins. Some examples:
+
+- If the "requirements.capabilities.name" property is set to "Mailbox" and the "requirements.capabilities.minVersion" to "1.10", then the add-in isn't installable on older versions of Office that don't support **Mailbox** requirement sets greater than version 1.9.
+- If the "requirements.scopes" is set to "mail", then the add-in is installable only on Outlook.
+- If the "requirements.formFactors" is set to only "desktop", then the add-in isn't installable on Office running on a mobile device.
+
+You can have more than one capability object. The following example shows how to ensure that an add-in is installable only on versions of Office that support two different requirement sets and not on mobile devices.
+
+```json
+"extensions": [
+    ...
+    "requirements": {
+        "capabilities": [
+            {
+                "name": "Mailbox",
+                "minVersion": "1.10"
+            },
+            {
+                "name": "DialogAPI",
+                "minVersion": "1.2"
+            }
+        ],
+        "formFactors": [
+            "desktop"
+        ]
+    }
+]
+```
+
+## Filter features
+
+The "requirements" properties in descendant objects of "extensions" are used to filter out (that is, block) some features of an add-in while still allowing the add-in to be installed. The implementation of this filtering is done at the source of installation, such as [AppSource](/office/dev/store/submit-to-appsource-via-partner-center) or [Microsoft 365 Admin Center](../publish/publish.md). If the version of Office doesn't support the requirements specified for the feature, then the JSON node for the feature is removed from the manifest before it is installed in the Office application. 
+
+### extensions.alternates.requirements
+
+The "extensions.alternates" property enables add-in developers to do the following:
+
+- Maintain a version of an add-in that was built on an older extensibility platform (such as COM or VSTO add-ins) or using the XML manifest, in addition to the version that uses the unified manifest.
+- Either hide or give preference to the version that uses the older technology.
+- Specify icons that are needed to make the unified manifest version of the add-in installable on Office versions that don't directly support the unified manifest. 
+
+[!INCLUDE [non-unified manifest clients note](../includes/non-unified-manifest-clients.md)]
```

Comment:
I don't think you can refer to another repo's include files. When you move this article over to the other repo, you might need to add a copy of this include file. Or copy the include content here if it won't be shared with other articles in that repo.